Content.Server/Kudzu/SpreaderSystem.cs

```diff
@@ -102,7 +102,7 @@ private bool TryGrow(EntityUid ent, TransformComponent? transform = null, Spread
         {
             var direction = (DirectionFlag) (1 << i);
             var coords = transform.Coordinates.Offset(direction.AsDir().ToVec());
-            if (grid.GetTileRef(coords).Tile.IsEmpty || _robustRandom.Prob(spreader.Chance)) continue;
+            if (grid.GetTileRef(coords).Tile.IsEmpty || _robustRandom.Prob(1 - spreader.Chance)) continue;
```

Comment:
uhh this seems like a logic change

Reply:
It is but it's also correcting a bug in said logic so meh.

Reply:
> uhh this seems like a logic change

yes, but:
- what moony said, the logic was bugged
- only the default on the prototype was being used anyways, which i changed to work with this new logic
- and bc of the second point: i found no usages of this in yaml, else i'd have changed those too ofc